### DISCUSSION TITLE
Customized Excel/CSV template (Kimai V2)

### DISCUSSION THREAD

**Discussion by lukasgranatier** [kimai/kimai#5183] [General]
Following this topic - https://github.com/kimai/kimai/discussions/3121, which was posted here on 02/2022 (before V2), I would like to ask if it is already possible to create custom Excel/CSV templates in addition to PDF and HTML.

In the documentation there is still only information about these 2 formats mentioned above:
https://www.kimai.org/documentation/export.html

Has anything changed since then or is the solution just the same as you recommend in the original topic?
Thank you for your reply.

**Comment by kevinpapst** [comment]
Nothing changed. Stil one issue on the roadmap.
